EngineLayer/Mods/Mods.txt

```diff
@@ -155,7 +155,7 @@ TG   K
 PP   Anywhere.
 MT   Common Biological
 DI   HCD:169.110323533
```

Comment:
Good catch on this. C4H7O2 looks right.

Will this change the `DI` mass? Should that be updated? I'm not sure how that's calculated.